src/Microsoft.Content.Build.MerfToSDPForJava.DataContracts/ServiceMapping/ServiceMapping.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.DataContracts
```

Comment:
how are we going to use "ServiceMapping" please?

Reply:
Remove